src/main/java/impl/DataProcessorImpl.java

```diff
@@ -0,0 +1,24 @@
+package impl;
+
+import java.util.List;
+import model.FruitTransaction;
+import service.DataProcessor;
+import strategy.OperationHandler;
+import strategy.OperationStrategy;
+
+public class DataProcessorImpl implements DataProcessor {
+    private OperationStrategy operationStrategy;
+
+    public DataProcessorImpl(OperationStrategy operationStrategy) {
+        this.operationStrategy = operationStrategy;
+    }
+
+    @Override
+    public void process(List<FruitTransaction> fruitTransactions) {
+        for (FruitTransaction transaction : fruitTransactions) {
+            OperationHandler operationHandler = operationStrategy
+                    .getHandler(transaction.getOperation());
+            operationHandler.apply(transaction);
+        }
+    }
+}
```

Comment:
According to the checklist, the class should not begin with an empty line. Please remove the empty line at the beginning of the class implementation.